WpfAnimatedGif/ImageAnimationController.cs

```diff
@@ -113,6 +113,15 @@ public void Pause()
         }
 
         /// <summary>
+        /// Stops and resets the animation.
+        /// </summary>
+        public void Stop()
+        {
+            _clockController.Begin();
```

Comment:
Calling `Begin()` to stop the animation feels a bit unnatural... especially if it's just to call `Pause()` afterwards. If you just want to rewind the animation, why not use `GotoFrame(0)` instead?